devtools/make_packages

```diff
@@ -0,0 +1,130 @@
+#!/bin/env python
+"""Caliopen package creation logic.
+
+Read package definition from a yaml file and build the corresponding
+package depending on language.
+
+cordova:
+  lauguage: javascript
+  description: an app packaged for android, ios & windows phone
+  dependencies:
+    nodejs: "^6.10"
+  packages:
+  - "/dist/android-app"
+  - "/dist/ios-app"
+  - "/dist/windows-phone-app"
+
+# Python packages
+caliopen_storage:
+  language: python
+  description: Caliopen python storage package
+  dependencies:
+    python: "^2.7"
+  packages:
+  - "/dist/python"
+
+"""
+
+import os
+import sys
+import subprocess as sp
+import argparse
+
+import logging
+import yaml
+
+log = logging.getLogger(__name__)
+logging.basicConfig()
+log.setLevel(logging.DEBUG)
+
+
+PYTHON_BUILD_CMD = 'python setup.py bdist_wheel --universal --dist-dir {}'
+JS_BUILD_CMD = 'yarn release:web:{}'
```

Comment:
actually this command pattern might not work shortly, I will have something like:

* `yarn run release:web` (does not exist yet. And `release:web:server` and `release:web:browser` build only one package)
* `yarn run release:app:ios`
* `yarn run release:app:android`
* `yarn run release:app:windows`
...

Also all build commands cannot be launched on one machine only: I don't think we can build for android, ios and windows on Linux for example.

Today, `yarn run release` build all available packages in one time, but I have to change it for this previous reason.

And later my goal is to split apps by package or technology in `frontend` directory e.g.:

```
src/frontend/
  + web/
  + cordova/
  + electron/
```